src/test/ui/type-alias-impl-trait/auxiliary/collect_hidden_types.rs

```diff
@@ -0,0 +1,21 @@
+#![feature(type_alias_impl_trait)]
+
+// edition:2018
```

Comment:
Similarly, I think this probably should be at the top of the file, I'm not sure it gets picked up by current compiletest otherwise.